### ISSUE TITLE
git: 'credential-manager-core' is not a git command. See 'git --help'.

### ISSUE THREAD

**Issue by Epi17** [CorrelAid/git-newbie-playground#5]
Hi,

I am trying to set up Git on my computer following your great tutorial! However, the I received the following error message when I tried to use the credential manager.  
git: 'credential-manager-core' is not a git command. See 'git --help'.
What should I do? I would highly appreciate any advice!

Best,
Epi17

**Comment by Susanne23** [comment]
Hi,
I have the same problem. According to https://github.com/git-ecosystem/git-credential-manager/issues/144 there should be a git-credential-manager-core.exe file in mingw64/libexec. But this file doesn't exist there nor anywhere else with the newest available Git version for Windows, although I selected the recommended choice regarding credential manager during installation.
So I'm also stuck here. 
Thanks for any help, best regards,
Susanne

**Comment by Susanne23** [comment]
But may be this is not needed anymore, see issue #3?

**Comment by Epi17** [comment]
I tried to run all instructions without the line git credential-manager-core configure. I am not sure whether this makes sense but I am still faced with the same error message.

**Comment by Susanne23** [comment]
Yes, same here. May be there is something missing in the recent Git package?
But it seems to work with entering userId/pwd (but still getting the git: 'credential-manager-core' is not a git command message when using git push ), and for me that's ok for now..
